mindsdb/integrations/handlers/jira_handler/README.md

```diff
@@ -0,0 +1,43 @@
+# Jira Handler
+
+This is the implementation of the Jira handler for MindsDB.
+
+## Jira
+In short, Jira is a tool to track the progress of software defects,story and releases.
+In this handler. python client of api is used and more information about this python client can be found (here)[https://pypi.org/project/atlassian-python-api/]
+
+
+## Implementation
+This handler was implemented using `duckdb`, a library that allows SQL queries to be executed on `pandas` DataFrames.
+
+In essence, when querying a particular table, the entire table will first be pulled into a `pandas` DataFrame using the Jira API. Once this is done, SQL queries can be run on the DataFrame using `duckdb`.
+
+Note: Since the entire table needs to be pulled into memory first (DataFrame), it is recommended to be somewhat careful when querying large tables so as not to overload your machine.
+
+The required arguments to establish a connection are,
+* `table_name`: Sample Jira table name.
+* `jira_url`: Jira  url
+* `user_id`: Jira User ID.
+* `api_key`: API key for the Jira API.
+* `jira_query`: Jira Search Query
+
+## Usage
+In order to make use of this handler and connect to an Access database in MindsDB, the following syntax can be used,
```

Comment:
Please change the `Access` database part

Reply:
Thanks for letting me know, I changed it